python/cuml/linear_model/linear_regression.pyx

```diff
@@ -260,7 +260,8 @@ class LinearRegression(LinearPredictMixin,
     intercept_ = CumlArrayDescriptor(order='F')
 
     @device_interop_preparation
-    def __init__(self, *, algorithm='eig', fit_intercept=True, normalize=False,
+    def __init__(self, *, algorithm='eig', fit_intercept=True,
+                 normalize=False, copy_X=True,
```

Comment:
Please add the `copy_X` parameter to the documentation and to the `get_param_names()` function.